.github/workflows/test.yml

```diff
@@ -52,11 +52,11 @@ jobs:
       #       -destination 'platform=macosx' \
       #       test
 
-      - name: tvOS Tests
-        run: |
-          xcodebuild \
-            -workspace Amplitude.xcworkspace \
-            -scheme Amplitude_tvOS \
-            -sdk appletvsimulator \
-            -destination 'platform=tvOS Simulator,name=Apple TV' \
-            test
+#      - name: tvOS Tests @TODO Fix flaky tvOS tests and re-enable
```

Comment:
Curious what's the error earlier for tvOS?

Reply:
Here is the [link](https://github.com/Mercy811/Amplitude-iOS/actions/runs/7660637761/job/20878345037?pr=5) of failed test mentioned in the description.

Reply:
https://stackoverflow.com/questions/52455652/xcode-10-seems-to-break-com-apple-commcenter-coretelephony-xpc This might help